test.ts

```diff
@@ -40,27 +44,58 @@ function itShouldValidateTestCase(testCase: TestCase, queriesDir: string, lapisP
         if (testCase.expectedStatusCode) {
             expect(response.status).to.equal(testCase.expectedStatusCode, actualResponseText);
         }
-        if (testCase.expectedResponse && testCase.expectedResponse.fileName) {
-            const expectedResponseFile = path.join(queriesDir, testCase.expectedResponse.fileName);
-            const expectedResponseText = await getExpectedResponseString(
-                expectedResponseFile,
-                testCase.expectedResponse.decompressFile,
-            );
+        if (testCase.expectedResponse) {
```

Comment:
Why do we actually need this `if`? The type says that we always have an `expectedResponse` (which also logically makes sense)

Reply:
True, initially it was optional